docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -2,29 +2,29 @@
 share information with colleagues.
 
 [float]
 [[detection-view-and-filter-alerts]]
 === View and filter detection alerts
-The Detections page offers various ways for you to organize and triage detection alerts as you investigate suspicious events. You can:
+The Alerts page offers various ways for you to organize and triage detection alerts as you investigate suspicious events. You can:
 
 * Filter for a specific rule in the KQL bar (for example,
 `signal.rule.name :"SSH (Secure Shell) from the Internet"`).
 
 NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the
-*Detections* and *Rule details* pages, and in Timeline when either `All` or
+*Alerts* and *Rule details* pages, and in Timeline when either `All` or
 `Detection alerts` is selected.
 
-* Use the date and time filter to select a time range that you’re interested in exploring. By default, this filter is set to search the last 24 hours.
-* View detection alerts generated by a specific rule. To do this, click *Manage detection rules*, then click on a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram.
+* Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
+* To view detection alerts generated by a specific rule. To do this, go to *Detect* -> *Rules*, and then select a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram. Alternatively, you can go to *Detect* -> *Alerts* -> *Manage detection rules*, and then click on a rule name in the All rules table to view the alerts associated with the rule.
 * Use the *Stack by* drop-down in the Trend histogram to select specific parameters to visualize the individual counts. For example, if you select `signal.rule.name`, the histogram displays the total counts by alert name.
-* Filter alert results to include building block alerts or to only show alerts from indicator match rules by selecting the *Additional filters* drop-down. By default, building block alerts are excluded from the Alerts table; therefore, including them expands the number of alerts.
+* To filter alert results to include building block alerts or to only show alerts from indicator match rules, select the *Additional filters* drop-down. By default, building block alerts are excluded from the Alerts table; therefore, including them expands the number of alerts.
 
 NOTE: When updating alert results to include building block alerts, the Security app searches the `.siem-signals-<Kibana space>` index for the `signal.rule.building_block_type` field. When looking for alerts created from indicator match rules, the app searches the same index for the `signal.rule.threat_mapping` field.
```

Comment:
Revise to "Security Solution" or maybe "Elastic Security"?
```suggestion
NOTE: When updating alert results to include building block alerts, the Security Solution searches the `.siem-signals-<Kibana space>` index for the `signal.rule.building_block_type` field. When looking for alerts created from indicator match rules, the Security Solution searches the same index for the `signal.rule.threat_mapping` field.
```

Reply:
Keeping for now since we use "app" elsewhere in our docs. :)